components/event-feed-service/pkg/persistence/feed-elastic.go

```diff
@@ -150,13 +153,23 @@ func (efs ElasticFeedStore) GetFeed(query *feed.FeedQuery) ([]*feed.FeedEntry, i
 		mainQuery = mainQuery.Must(rangeQuery)
 	}
 
-	searchService := efs.client.Search().
+	searchSource := olivere.NewSearchSource().
```

Comment:
These changes allow the elasticsearch query to be logged for testing.

Reply:
ooh, this is way nicer than a 100 line comment with an example